### PR TITLE
- Clarifications for Public Cloud setup

### PR DESCRIPTION
**Pull request by rjschwei**
# Description

Current information is partially incorrect and partially misleading

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/installation/pages/pubcloud-setup.adoc [hunk 1]
```diff
@@ -1,20 +1,23 @@
 [[public-cloud-setup]]
 = Public Cloud Setup
 
-Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
-However, you must have the correct {productname} and {sle} entitlements to receive updates before you can log in.
+{productname} is pre-installed in images available in the Public Cloud. You do not need to perform any installation steps.
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:4]:
```suggestion
{productname} is pre-installed on images that are available in the Public Cloud. 
You do not need to perform any installation steps.
```

File: modules/installation/pages/pubcloud-setup.adoc [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+However, you must have the correct {productname} and {sle} entitlements to receive updates before you can manage client systems.
 
-For most bring your own service (BYOS) images, these entitlements are provided by {suse}, and can be obtained using the {scc}.
-If you are using an on demand image, you need to do some extra preparation before you can register to {scc}.
-
-When you have registered, all {sle} modules are activated.
-You can then activate the public cloud module.
+{productname} is only offered as bring your own subscription (BYOS) images. Entitlements are provided by {suse} or a partner, and can be obtained using the {scc}.
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:7]:
```suggestion
{productname} is only offered as bring your own subscription (BYOS) images. 
Entitlements are provided by {suse} or a partner, you can use {scc} to obtain your entitlements.
```

File: modules/installation/pages/pubcloud-setup.adoc [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+{sle} system images for clients are generally available in 2 flavors, on-demand images (also referred to as pay as you go - PAYG) and BYOS images. If you are managing on-demand instances the entitlement for the client itself is part of the on-demand pricing. For on-demand instances you need to do some extra preparation before you can manage the system with {productname}.
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:8]:
```suggestion
{sle} system images for clients are generally available as on-demand or pay as you go (PAYG) images, or as bring your own subscription (BYOS) images. 
If you are managing on-demand instances, the entitlement for the client is part of the on-demand pricing. 
For on-demand instances you need to do some extra preparation before you can manage the system with {productname}.
```

File: modules/installation/pages/pubcloud-setup.adoc [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
 
+An instance started from a SUSE published on-demand instance will automatically register against and update infrastructure and all {sle} modules are activated. This automation should be disabled for systems managed with {productname}.
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:10]:
```suggestion
An instance started from an on-demand image provided by {suse} is automatically registered, and the update infrastructure and {sle} modules are activated. 
For {productname} clients, you need to disable this automation before you begin.
```

File: modules/installation/pages/pubcloud-setup.adoc [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
 
 
 .Procedure: Preparing On Demand Images
-. Log in the on demand instance.
-. At the command prompt, as root, remove the `cloud-regionsrv-client` package:
+. Log into the on demand instance.
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:14]:
```suggestion
. Log in to the on-demand instance.
```

File: modules/installation/pages/pubcloud-setup.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/installation/pages/pubcloud-setup.adoc:8]: We probably need to check this entire chapter for hyphenation consistency in "on-demand".

File: modules/installation/pages/pubcloud-setup.adoc [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
+. At the command prompt, as root, remove the registration data and repositories
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:15]:
```suggestion
. At the command prompt, as root, remove the registration data and repositories:
```

File: modules/installation/pages/pubcloud-setup.adoc [hunk 7]
```diff
@@ -1,20 +1,23 @@
...
++
+---
+registercloudguest --clean
+---
+. then remove the `cloud-regionsrv-client` package:
```
Comment (Loquacity) [inline modules/installation/pages/pubcloud-setup.adoc:20]:
```suggestion
. Remove the ``cloud-regionsrv-client`` package:
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
Thanks for this, it was sorely needed! Just some edits for writing style.

**Review by keichwa — APPROVED** [review]

**Pull request merged**